### PR TITLE
4244 - Add more mixins

### PR DESCRIPTION
**Pull request by jmacaluso711**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**
Adding some position mixins

**Related github/jira issue (required)**:
related to #4244

**Steps necessary to review your pull request (required)**:
- pull branch
- `npm run build:mixins`
- check the dist folder and make sure the mixins build properly

### CODE REVIEW

File: scripts/node/build-mixins.js [hunk 1]
```diff
@@ -42,11 +44,15 @@ const propKeys = {
   minWidth: 'min-w',
   padding: 'p',
   placeholderOpacity: 'placeholder-opacity',
+  position: 'position',
+  rigth: 'right',
```
Comment (EdwardCoyle) [inline scripts/node/build-mixins.js:48]:
```suggestion
  right: 'right',
```

**Review by EdwardCoyle — APPROVED** [review]
:banana: Great additions, just found one thing below

**Review by tmcconechy — CHANGES_REQUESTED** [review]
@jmacaluso711 which mixins does this add? I think that the idea is that the spacing setting should add these properties.

Was trying to keep the ui.config.js file compatible with tailwind in case someone wanted it.  And some of the random keys are just added https://github.com/infor-design/design-system/blob/3.0.x/scripts/node/build-mixins.js#L201 https://github.com/tailwindlabs/tailwindcss/blob/master/stubs/defaultConfig.stub.js

Can you list the mixins desired/added and we can work though the best way to add them?

**Comment by tmcconechy** [comment]
Ok cool. Yeah i think that would be nice. Not sure this makes it incompatible those probably would be ignored.

- for position: add the rules in directly here https://github.com/infor-design/design-system/blob/3.0.x/scripts/node/build-mixins.js#L201
- for left, right, bottom, top? maybe this is fine what i had done is making the spacing generate the margins ect... https://github.com/infor-design/design-system/blob/3.0.x/scripts/node/build-mixins.js#L133 but looking again this is more convoluted that the config.

So maybe this is ok for now

**Comment by jmacaluso711** [comment]
@tmcconechy Does anything need to change in this PR?

**Comment by tmcconechy** [comment]
Lets just go with this for now.

**Review by tmcconechy — APPROVED** [review]

**Pull request merged**